pennylane/ops/cv.py

```diff
@@ -770,15 +770,15 @@ class FockDensityMatrix(CVOperation):
 
     **Details:**
 
-    * Number of wires: None (applied to the entire subsystem)
+    * Number of wires: Any
     * Number of parameters: 1
     * Gradient recipe: None (uses finite difference)
 
     Args:
         state (array): a single mode matrix :math:`\rho_{ij}`, or
             a multimode tensor :math:`\rho_{ij,kl,\dots,mn}`, with two indices per mode
     """
-    num_wires = 0
+    num_wires = Any
     num_params = 1
     par_domain = "A"
     grad_method = "F"
```

Comment:
same comment about gradient method. Should it not be `None`?

Reply:
Can you kindly check this for all state prep methods? Most of them have `grad_method = "F"` 
I am not sure why this is so and if this is correct. Thanks

Reply:
See https://github.com/XanaduAI/pennylane/pull/277#discussion_r307989136.

If you would like, feel free to make an issue regarding cleaning up the gradient recipes for state preparation :slightly_smiling_face:

Reply:
ok thanks, I will.